### PR TITLE
Very basic PagerDuty integration

### PR DESCRIPTION
**Pull request by stkem**
cc @eishay @andrewconner @raymondng

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/common/healthcheck/AirbrakeNotifier.scala [hunk 1]
```diff
@@ -95,12 +99,38 @@ class AirbrakeSender @Inject() (
...
 
+class PagerDutySender @Inject() (httpClient: HttpClient) {
+  import play.api.libs.concurrent.Execution.Implicits.defaultContext
+
+  def openIncident(description: String, exception: Throwable): Unit = {
+    val payload = Json.obj(
+      "service_key"  -> "7785f2cc14ec44e49ae3bb8186400cc7",
+      "event_type"   -> "trigger",
+      "description"  -> description.take(1000),
+      "incident_key" -> description.take(1000),
```
Comment (eishay) [inline kifi-backend/modules/common/app/com/keepit/common/healthcheck/AirbrakeNotifier.scala]: how about using the `error.signature` for that? `AirbrakeErrorSignature.value` is a string

**Comment by andrewconner** [comment]
How does dev mode handle this? I couldn't find the bindings.

**Comment by stkem** [comment]
@andrewconner Dev mode is not hooked up to Pagerduty. All the `panic` endpoints are faked out in `AirbrakeModule.scala` for Dev.

File: kifi-backend/modules/common/app/com/keepit/common/healthcheck/AirbrakeNotifier.scala (hunk 1 above)
Comment (stkem) [inline kifi-backend/modules/common/app/com/keepit/common/healthcheck/AirbrakeNotifier.scala]: Didn't know that was a thing. Changing it.

**Comment by eishay** [comment]
lgtm

**Comment by eishay** [comment]
lets get it out and see how it works in prod

**Pull request merged**